Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'activefedora-aggregation', github: 'projecthydra-labs/activefedora-aggregation', branch: 'master'
+gem 'activefedora-aggregation', github: 'projecthydra-labs/activefedora-aggregation', branch: 'FAIL_to_add_item'
```

Comment:
This branch has tests passing.  Once that branch is committed to master, I will switch it to the master branch.  Temporary kludge to have tests passing as a bench mark for identifying changes that break the code or introduce new errors.

How close are we to having that branch committed to master?